tower-balance/src/load/peak_ewma.rs

```diff
@@ -307,8 +308,8 @@ fn nanos(d: Duration) -> f64 {
 
 #[cfg(test)]
 mod tests {
-    extern crate tokio_executor;
-    extern crate tokio_timer;
+    use tokio_executor;
```

Comment:
same here?

Reply:
not to the best of my knowledge—it's certainly nice, but this is a large diff as-is and i'd be happy to address in subsequent PRs.